README.md

```diff
@@ -120,3 +120,39 @@ Usage
 -----
 
 To be written.
+
+Spatial Function Reference
+--------------------------
+
+This is a list of all functions which are are currently implemented in the geo project. Some functions are only available
+if you use a specific geometry engine.
+
+| Function Name | GEOS | PostGIS | MySQL | SpatiaLite | OpenGIS standard |
+|---------------|------|---------|-------|------------|------------------|
+|`area`         |  X   |    X    |   X   |     X      |        X         |
+|`boundary`     |  X   |    X    |       |     X      |        X         |
+|`buffer`       |  X   |    X    |       |     X      |        X         |
```

Comment:
Just tested it, and `ST_Buffer()` is available as well, it's just not documented yet as far as I can see, so let's include it here!